pom.xml

```diff
@@ -719,7 +721,7 @@
                     <configuration>
                         <rules>
                             <requireJavaVersion>
-                                <version>[1.8,1.9]</version>
+                                <version>[1.8,12]</version>
```

Comment:
why is this 12, not 11?

Reply:
because I wanted all versions of 11 to be in range, I think that's a boundary marker.

Reply:
would `[8,12)` work to make it an open interavl?

Reply:
[8,12] usually implies that it includes 12.